Readme.md

```diff
@@ -1,14 +1,15 @@
 # WebVI Examples for G Web Development Software
 
 This repository has examples for:
-**G Web Development Software 2021**
+**G Web Development Software 2022**
 
 For LabVIEW NXG versions see the [previous releases](https://github.com/ni/webvi-examples/releases/)
 
 The following examples demonstrate different ways of using WebVIs. Each directory contains example code and a `README.md` file explaining the example.
+To understand the basics of programming with G language refer to the examples in [Programming Basics directory](ProgrammingBasics).
```

Comment:
It might be better to keep this in a separate paragraph from the reference to existing WebVI examples.